src/Types.hs

```diff
@@ -319,12 +320,14 @@ newline :: RichText
 newline = "\n"
 
 showRecipe :: (String -> String) -> Recipe -> Maybe Int -> RichText
-showRecipe t r maybeServings =
-  case maybeServings of
-    Nothing       -> "Error: use a serving size greater than 0\n"
-    Just servings -> showRecipeHeader t r (Just servings)
-                    ~~ newline ~~ List.unlines (showRecipeSteps r)
-
+showRecipe t r maybeServings = 
+    showRecipeHeader t r serv
```

Comment:
Instead of the `where` with the `case`, you can equivalently use 
```suggestion
    showRecipeHeader t r (maybeServings <|> Just 1)
```
where `<|>` comes from `Control.Applicative`.